### ISSUE TITLE
[DEV] Prevent duplicate projects from appearing in the Projects section of the User Profile

### ISSUE THREAD

**Issue by pyxld-kris** [dev-launchers/dev-launchers-platform#896]
Currently, if a member is both a leader and a member of a project then the project gets listed twice in the Projects section of the user profile. These projects should be listed only once.

![image.png](https://images.zenhubusercontent.com/633e1a5b90f8260b0fcb1579/203ccf0f-3cf8-4cf7-ab79-d818c6a49925)
_Image showing a project being listed twice_

**Acceptance Criteria**
 - The Projects section of the User Profile lists all of the projects a member is assigned to as either a leader or a member
 - The Projects section only lists each project once

**Comment by KaitlynMallare** [comment]
No longer needed - deleting/closing as not planned
